backend/templates/contract_template.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+
+<!--NB there also exist a react version of the contract frontend, in case of changes both must be updated.-->
+
+<html lang="no">
+    <head>
+    <title>Leiekontrakt</title>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="" />
+    <style>
+      span {
+        white-space: nowrap;
+      }
+    </style>
+    </head>
+    <body style="font-family: Montserrat, serif">
```

Comment:
The font used in the templates was changed in #299 to Open Sans, and sans-serif as a fallback font.
```suggestion
    <body style="font-family: Open Sans, sans-serif">
```